### PR TITLE
Hotfix - Remove tap highlight on toggle components

### LINKED ISSUE #399: Remove tap highlight on mobile devices for toggles
**Describe the bug**
On mobile devices, when pressing the toggle component, there is tap highlight.

**To Reproduce**

1. Go to **Toggle component**
2. Click on any toggle while being on mobile (or in the browser device emulation)
3. There is a highlight when pressing the toggle

**Expected behavior**
Like in desktop / tablet, I think there shouldn't be a highlight when pressing the toggle on mobile devices

**Screenshots**

https://github.com/markmead/hyperui/assets/46305144/605f3821-e77b-4709-b50c-363068869354

**Smartphone (please complete the following information):**

- Device: iPhone XR
- OS: iOS 17
- Browser safari
- Version 17

### PR DESCRIPTION
**Pull request by NDOY3M4N**
This closes #399

### CODE REVIEW

File: public/components/application-ui-toggles/1-interactive.html [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+  }
+-->
+
 <script defer src="https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js"></script>
 
 <label
   x-data="{ isChecked: false }"
   for="AcceptConditions"
-  class="relative h-8 w-14 cursor-pointer"
+  class="relative h-8 w-14 cursor-pointer [-webkit-tap-highlight-color:_transparent]"
```
Comment (markmead) [inline public/components/application-ui-toggles/1-interactive.html:14]: It doesn't matter too much but I'd rather see this as `[&::-webkit-tap-highlight-color:_transparent]`.

File: public/components/application-ui-toggles/1-interactive.html [hunk 2]
```diff
@@ -1,9 +1,17 @@
+<!--
+  @layer components {
+    .no-tap-highlihgt {
+      -webkit-tap-highlight-color: transparent;
+    }
+  }
+-->
```
Comment (markmead) [inline public/components/application-ui-toggles/1-interactive.html:7]: No need for these comments.

**Review by markmead — CHANGES_REQUESTED** [review]

File: public/components/application-ui-toggles/1-interactive.html (hunk 2 above)
Comment (NDOY3M4N) [reply public/components/application-ui-toggles/1-interactive.html:7]: Ok will remove them

File: public/components/application-ui-toggles/1-interactive.html (hunk 1 above)
Comment (NDOY3M4N) [reply public/components/application-ui-toggles/1-interactive.html:14]: I've tried this in [here](https://play.tailwindcss.com/VXZcs4P3WO) but the styles are not applied.

**Review by markmead — APPROVED** [review]
LGTM!

**Pull request merged**